ui/src/app/sensors/components/sensor-side-panel.tsx

```diff
@@ -0,0 +1,56 @@
+import {SlidingPanel, Tabs} from 'argo-ui';
+import * as React from 'react';
+import {useState} from 'react';
+import {Sensor} from '../../../models';
+import {Node} from '../../shared/components/graph/types';
+import {EventsPanel} from '../../workflows/components/events-panel';
+import {SensorLogsViewer} from './sensor-logs-viewer';
+
+export const SensorSidePanel = ({
+    isShown,
+    namespace,
+    sensor,
+    selectedTrigger,
+    onTriggerClicked,
+    onClose
+}: {
+    isShown: boolean;
+    namespace: string;
+    sensor: Sensor;
+    selectedTrigger: string;
+    onTriggerClicked: (selectedNode: string) => void;
+    onClose: () => void;
+}) => {
+    const queryParams = new URLSearchParams(location.search);
+    const [logTab, setLogTab] = useState<Node>(queryParams.get('logTab'));
+
+    return (
+        <SlidingPanel isShown={isShown} onClose={onClose}>
+            {!!sensor && (
+                <div>
+                    <h4>
+                        Sensor/{sensor.metadata.name}
```

Comment:
If we are in the sensor view, we know we're looking at sensors. Not sure if the `Sensor/` is necessary